src/Bundler.js

```diff
@@ -37,15 +37,13 @@ class Bundler extends EventEmitter {
     this.delegate = options.delegate || {};
     this.bundleLoaders = {};
 
-    this.addBundleLoader(
-      'wasm',
-      require.resolve('./builtins/loaders/wasm-loader')
-    );
-    this.addBundleLoader(
-      'css',
-      require.resolve('./builtins/loaders/css-loader')
-    );
-    this.addBundleLoader('js', require.resolve('./builtins/loaders/js-loader'));
+    const loadersPath = `./builtins/loaders/${
```

Comment:
would be cool to add support for registering bundle loaders for specific environments. this would allow third party plugins to do the same:

```javascript
bundler.addBundleLoader('wasm', {
  node: require.resolve('./node/wasm-loader'),
  browser: require.resolve('./browser/wasm-loader')
});
```

Want to make a followup PR to support that?